app/models/concerns/foreman/thread_session.rb

```diff
@@ -37,6 +37,24 @@ def clear_thread
       end
     end
 
+    # This allows getting and setting all current values in case it's needed,
```

Comment:
OK then. Created a refactor [issue](http://projects.theforeman.org/issues/20390) in redmine, just to make it visible.